lib/packages.js

```diff
@@ -5,6 +5,7 @@ var features = JSON.parse(featuresJson).features;
 var packages = {
   'container':                  {trees: null,  requirements: []},
   'ember-environment':          {trees: null,  requirements: [], skipTests: true},
+  'ember-console':               {trees: null,  requirements: [], skipTests: true},
   'ember-metal':                {trees: null,  requirements: ['ember-environment'], vendorRequirements: ['backburner']},
```

Comment:
I believe you need to add `ember-logger` to the list of `requirements` here also.